samples/hotel-booking-agent/agent/.env.example

```
@@ -0,0 +1,11 @@
+OPENAI_API_KEY=
+OPENAI_MODEL=
+
+HOTEL_API_BASE_URL=
+
+PINECONE_API_KEY=
+PINECONE_SERVICE_URL=
+PINECONE_INDEX_NAME=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PINECONE_INDEX_NAME key should go before the PINECONE_SERVICE_URL key

(UnorderedKey)